.storybook/main.js

```diff
@@ -3,7 +3,12 @@ module.exports = {
     {
       directory: '../src/ui',
       titlePrefix: 'Design System/Components/',
-      files: '**/*.stories.*',
+      files: '*/*.stories.*',
+    },
+    {
+      directory: '../src/ui',
+      titlePrefix: 'Design System/Components/',
+      files: '*.stories.*',
```

Comment:
Added this as there were stories missing that were not inside any folder, e.g. `Modal`, `ToggleButton` among others.